.gitignore

```diff
@@ -26,6 +26,8 @@ npm-debug.log*
 yarn-debug.log*
 yarn-error.log*
 
+build-storybook.log
```

Comment:
This file is created if you publish a build locally (which requires you to know the token).